applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes_fractional.h

```diff
@@ -0,0 +1,546 @@
+    * Since the enrichment is performed elementwise, a purely local static condensation step is performed,
+    * meaning that no extra degrees of freedom are added..
+    *
+    * Since a jump in the pressure can be considered, the element shall be able to habdle moderate changes of the viscosity
+    * between the two fluids to be considered
+    * The main difference between this element and the generic one "TwoFluidNavierStokes" is that the Navier-Stokes momentum conservation equation is solved in two problems using a fractional splitting approach. This element refers to the second problem.
+    */
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+    template <class TElementData>
+    class TwoFluidNavierStokesFractional : public FluidElement<TElementData>
+    {
+    public:
+        /// Counted pointer of
+        KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(TwoFluidNavierStokesFractional);
+
+        ///@name Type Definitions
+        ///@{
+
+        typedef Node NodeType;
+        typedef Geometry<NodeType> GeometryType;
+        typedef Geometry<NodeType>::PointsArrayType NodesArrayType;
+        typedef Vector VectorType;
+        typedef Matrix MatrixType;
+        typedef std::size_t IndexType;
+        typedef std::size_t SizeType;
+        typedef std::vector<std::size_t> EquationIdVectorType;
+        typedef std::vector<Dof<double>::Pointer> DofsVectorType;
+        typedef PointerVectorSet<Dof<double>, IndexedObject> DofsArrayType;
+        typedef typename FluidElement<TElementData>::ShapeFunctionsType ShapeFunctionsType;
+        typedef typename FluidElement<TElementData>::ShapeFunctionDerivativesType ShapeFunctionDerivativesType;
+        typedef typename FluidElement<TElementData>::ShapeFunctionDerivativesArrayType ShapeFunctionDerivativesArrayType;
+        constexpr static unsigned int Dim = FluidElement<TElementData>::Dim;
+        constexpr static unsigned int NumNodes = FluidElement<TElementData>::NumNodes;
+        constexpr static unsigned int BlockSize = FluidElement<TElementData>::BlockSize;
+        constexpr static unsigned int LocalSize = FluidElement<TElementData>::LocalSize;
+        constexpr static unsigned int StrainSize = (Dim - 1) * 3;
+
+        ///@}
+        ///@name Life Cycle
+        ///@{
+
+        /// Default constuctor.
+        /**
+         * @param NewId Index number of the new element (optional)
+         */
+        TwoFluidNavierStokesFractional(IndexType NewId = 0);
+
+        /// Constructor using an array of nodes.
+        /**
+         * @param NewId Index of the new element
+         * @param ThisNodes An array containing the nodes of the new element
+         */
+        TwoFluidNavierStokesFractional(IndexType NewId, const NodesArrayType &ThisNodes);
```

Comment:
```suggestion
        TwoFluidNavierStokesFractional(
            IndexType NewId,
            const NodesArrayType &ThisNodes);
```
Please follow the style guide (also below).